src/aws.ts

```diff
@@ -53,15 +54,22 @@ export const getRegistriesData = async (registry: string, username?: string, pas
     authTokenRequest['registryIds'] = accountIDs;
   }
 
+  const credentials =
+    username && password
+      ? {
+          accessKeyId: username,
+          secretAccessKey: password
+        }
+      : undefined;
```

Comment:
You beat me to it, this should fix #129

Reply:
Yes! 😄 just saw test test branch :)

Reply:
Should allow it to fallback to the default credential chain that the aws-sdk uses